code/backtracking/Graph Coloring Problem/coloring_problem.cpp

```diff
@@ -0,0 +1,56 @@
+#include<iostream>

```

Comment:
Remove spaces from the folder name, if you can! Also, rename the file to _graph_ \_coloring\_problem.cpp